src/test/kotlin/com/igorwojda/integer/pyramidgenerator/challenge.kt

```diff
@@ -4,7 +4,14 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 fun generatePyramid(n: Int): List<String> {
-    TODO("not implemented")
+
+    val pyramidElementsList = mutableListOf<String>()
+    var lastRow = if (n == 2) n / 2 else (n - 1)
```

Comment:
I know it is tempting to write and if-else as a single line statement, but it will make your code easier to debug and read if you add curly braces for the if-else statements.